src/common/config_parser.cpp

```diff
@@ -637,19 +641,18 @@ void ConfigParser::parseOptions(int argc, char** argv, bool doValidate) {
     config_ = cli.getConfig();
   }
 
-  if(has("interpolate-env-vars")) {
+  if(hasBool("interpolate-env-vars")) {
     cli::ProcessPaths(config_, cli::InterpolateEnvVars, PATHS);
   }
 
-  if(has("relative-paths") && !has("dump-config")) {
+  if(hasBool("relative-paths") && !hasBool("dump-config")) {
```

Comment:
Is this correct? Are you really testing whether the option is present, or rather whether it is specified to be true? Then it would be more like

    get<bool>("relative-paths", false)